Library/Homebrew/cmd/commands.rb

```diff
@@ -1,8 +1,9 @@
 module Homebrew
   def commands
     # Find commands in Homebrew/cmd
+    with_directory = false
     cmds = (HOMEBREW_REPOSITORY/"Library/Homebrew/cmd").
-           children(with_directory=false).
+           children(with_directory).
```

Comment:
I don't agree; I presume the `with_directory` was added for argument readability (boolean arguments are terrible) and this preserves that.